lib/run.js

```diff
@@ -50,19 +53,19 @@ function run(config, done) {
                     total: tasks.length,
                     info: info,
                     testCmd: testCmd,
-                    installCmd: "npm i" + (config.registry ? (" --registry " + config.registry) : "")
+                    installCmd: (config.useYarn ? "yarn add" : "npm i") + (config.registry ? (" --registry " + config.registry) : "")
```

Comment:
I guess the `--registry` functionallity is the only one this PR isn't covering with yarn yet. Any ideas?